easy_rec/python/test/hive_input_test.py

```diff
@@ -0,0 +1,274 @@
+# -*- encoding:utf-8 -*-
+# Copyright (c) Alibaba, Inc. and its affiliates.
+"""Define cv_input, the base class for cv tasks."""
+import tensorflow as tf
+from easy_rec.python.input.hive_input import HiveInput
+from easy_rec.python.protos.dataset_pb2 import DatasetConfig
+from easy_rec.python.utils import test_utils
+from easy_rec.python.utils.config_util import *
+from easy_rec.python.utils.test_utils import *
+
+if tf.__version__ >= '2.0':
+  #tf = tf.compat.v1
+  import tensorflow.compat.v1 as tf
+
+gfile = tf.gfile
+
+if tf.__version__ >= '2.0':
+  from tensorflow.python.framework.ops import disable_eager_execution
+
+  disable_eager_execution()
+  tf = tf.compat.v1
+
+
+class HiveInputTest(tf.test.TestCase):
+
+  def __init__(self, methodName='HiveInputTest'):
+    super(HiveInputTest, self).__init__(methodName=methodName)
+    self._input_path = 'census_income_train_simple'
+
+  @RunAsSubprocess
+  def test_hive_input(self):
+    data_config_str = """
```

Comment:
long config maybe better to put into a config file

Reply:
it’s part of config file , test config style.